spec/workers/enqueue_subject_queue_worker_spec.rb

```diff
@@ -53,19 +60,42 @@
       end
     end
 
-    describe "#load_subjects" do
+    context "when selecting via cellect selection strategy" do
+      it 'should attempt to queue the selected set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.